### PR TITLE
Add region information for omf ingress

### PR DESCRIPTION
**Pull request by hansatosisoft**

### CODE REVIEW

File: Documentation/DataIngress/OMF_Ingress_Topics.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Topics
 Topic Information 
 -----------------------
 
-A Topic is used to aggregate data received from Clients. The Topic then makes the data available for consumption 
+A Topic aggregates data received from Clients. The Topic then makes the data available for consumption
```
Comment (mschellenger) [inline Documentation/DataIngress/OMF_Ingress_Topics.md]: I would make some note here that data sent to a topic is stored in that region. Maybe "A Topic aggregates data received from Clients in the region of it's Namespace" or something similar

File: Documentation/DataIngress/OMF_Ingress_Subscriptions.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ Subscriptions
 Subscription Information 
 -----------------------
 
-A Subscription is used to consume data from a Topic. Multiple Subscriptions can retrieve data from a single Topic.
+A Subscription consumes data from a Topic. Multiple Subscriptions can retrieve data from a single Topic. 
+
+A Subscription can consume data from a Topic in a different Namespace. However, the Topic's Namespace must be in the same Region as the Subscription's Namespace.
```
Comment (mschellenger) [inline Documentation/DataIngress/OMF_Ingress_Subscriptions.md]: Similar to below, I would mention that data that is forwarded to SDS from the subscription is stored in the same region as the namespace

File: Documentation/DataIngress/OMF_Ingress_Subscriptions.md [hunk 3]
```diff
@@ -7,14 +7,16 @@ Subscriptions
...
+
+A Subscription can consume OMF messages from a Topic in a different Namespace. However, the Topic's Namespace must be in the same Region as the Subscription's Namespace. OMF messages that the Subscription is processing are temporarily stored in the Region of its Namespace.
 
 The API calls in this section are used to create and manipulate Subscriptions.
 
 Sequential Data Store (Sds) Subscription 
 ---------------
 
-A Sequential Data Store Subscription retrieves data from a Topic and writes it directly to a namespace in the Sequential Data Store. Currently only Sds Subscriptions are supported. The documentation uses Sds Subscription and Subscription interchangeably.
+A Sequential Data Store Subscription retrieves OMF messages from a Topic and writes it directly to a Namespace in the Sequential Data Store. Currently only Sds Subscriptions are supported. The documentation uses Sds Subscription and Subscription interchangeably.
```
Comment (osimarkrogers) [inline Documentation/DataIngress/OMF_Ingress_Subscriptions.md]: Change "writes it" to "writes them"

Comment (hansatosisoft) [reply Documentation/DataIngress/OMF_Ingress_Subscriptions.md]: Good catch. Done.

**Pull request merged**